clients/da-vinci-client/src/main/java/com/linkedin/davinci/ingestion/main/MainIngestionMonitorService.java

```diff
@@ -302,11 +296,14 @@ private void resumeOngoingIngestionTasks() {
 
   private void collectIngestionServiceMetrics() {
     long requestTimestamp = System.currentTimeMillis();
-    if (!metricsClient.collectMetrics(isolatedIngestionProcessStats)) {
+    if (metricsClient.collectMetrics(isolatedIngestionProcessStats)) {
+      LOGGER.info(
+          "Metric request completed in {} seconds.",
+          TimeUnit.MILLISECONDS.toSeconds(System.currentTimeMillis() - requestTimestamp));
```

Comment:
Do we need a log in the successful path? Should we reduce it to debug-level? I suppose it's not a hot path, but it still seems a bit spammy? IDK for sure, just wondering.

Reply:
I was trying to keep a log to compare how much time metric collection spent on different cluster in EI. But maybe debug level can be enough as we can change the logging level on the host.